README.md

```diff
@@ -81,11 +81,11 @@ xhr.onreadystatechange = () => {
 | `setup()` | Replace the global `XMLHttpRequest` object with the `MockXMLHttpRequest`. |
 | `teardown()` | Restore the global `XMLHttpRequest` object to its original state. |
 | `reset()` | Remove all request handlers. |
-| `get(url: string | regex, callback)` | Create `GET` mock response for a specific url. |
-| `post(url: string | regex, callback)` | Create `POST` mock response for a specific url. |
-| `put(url: string | regex, callback)` | Create `PUT` mock response for a specific url. |
-| `patch(url: string | regex, callback)` | Create `PATCH` mock response for a specific url. |
-| `delete(url: string | regex, callback)` | Create `DELETE` mock response for a specific url. |
+| `get(url: string &#124; regex, callback)` | Create `GET` mock response for a specific url. |
```

Comment:
The table rendering is currently broken

Reply:
Ah that's a bummer! Just found out that they do [support escaping of special markdown characters](https://enterprise.github.com/downloads/en/markdown-cheatsheet.pdf) with a simple `\`. Works even in code blocks. We can make another PR later to fix these.

Reply:
Ah, awesome!